imports/plugins/included/product-variant/containers/productsContainer.js

```diff
@@ -166,9 +166,16 @@ function composer(props, onData) {
     window.prerenderReady = true;
   }
 
+  const shops = Shops.find({
```

Comment:
Maybe we could name this something like `activeShops` to make it clearer what we are filtering for below.

Reply:
If we filter for just the `_id` field we could also avoid using the `map` below.

Reply:
We'd still have to use map coz after:
```
const activeShopIdsArray = Shops.find({
    "workflow.status": {
      $in: ["active", "new"]
    }
  }, { fields: { _id: 1 } }).fetch();
```
It returns an array of objects that looks like this:
```
[ { _id: "J8Bhq3uTtdgwZx3rz" }, { _id: "Xw8zPBgTmSF7ZdFZS" } ]
```
So eventually I'd need to get just the values

Reply:
True. I forgot that it doesn't just return an array of ids.